digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/roadaddress/oracle/RoadAddressDAO.scala

```diff
@@ -6,14 +6,29 @@ import fi.liikennevirasto.digiroad2.Point
 import slick.jdbc.{GetResult, StaticQuery => Q}
 import com.github.tototoshi.slick.MySQLJodaSupport._
 import fi.liikennevirasto.digiroad2.asset.SideCode
+import fi.liikennevirasto.digiroad2.asset.SideCode.{AgainstDigitizing, BothDirections, TowardsDigitizing, Unknown}
```

Comment:
Yes, it's really necessary, if you remove some of them the code doesn't work any more.
For modify this, we need to change the code and I think it's not necessary because of a Import.